examples/amp-subscriptions.amp.html

```diff
@@ -334,12 +329,16 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
   </footer>
 
 
-  <template type="amp-mustache" subscriptions-dialog subscriptions-display="loggedIn">
+  <template type="amp-mustache" subscriptions-dialog subscriptions-display="data.loggedIn">
     Dialog for logged in users
   </template>
-  <template type="amp-mustache" subscriptions-dialog subscriptions-display="NOT loggedIn">
+  <template type="amp-mustache" subscriptions-dialog subscriptions-display="NOT data.loggedIn">
     <div>Dialog for non logged in users<div>
-    <div subscriptions-display="NOT subscribed">btw u are not subscribed</div>
+    <div subscriptions-display="NOT data.loggedIn"
+      subscriptions-action="subscribe"
+      subscriptions-service="subscribe.google.com"
+      subscriptions-decorate>Login</div>
+    <div subscriptions-display="NOT grantReason">btw u are not subscribed</div>
```

Comment:
Let's do `grantReason != "SUBSCRIBED"` (SUBSCRIBER, whichever is the latest on this.)

Reply:
reverted this and loggedIn change as that stuff is in PR #15059